lemur/common/celery.py

```diff
@@ -145,10 +145,23 @@ def report_celery_last_success_metrics():
     schedule = current_app.config.get("CELERYBEAT_SCHEDULE")
     for _, t in schedule.items():
         task = t.get("task")
-        last_success = int(red.get(f"{task}.last_success") or 0)
+
+        metric_tags = {}
+        kwargs = t.get("kwargs")
+        if kwargs is not None and "source" in kwargs:
+            source = kwargs["source"]
+            last_success = int(red.get(f"{source}.last_success") or 0)
```

Comment:
Is it worth keying this using `{task}_{source}` while we're in here to prevent collisions with future tasks which might take `source` as an argument?

Reply:
yes thats a good idea, let me do that